.github/workflows/build-docker-images.yml

```diff
@@ -0,0 +1,115 @@
+name: build-docker-images
+
+on:
+  pull_request:
+#  workflow_dispatch:
+
+env:
+  DOCKER_BUILDKIT: 1
+  REGISTRY: docker.elastic.co
+  PREFIX: observability-ci
+
+jobs:
```

Comment:
This workflow works only on feature branches due to it uses SECRETS, it is fine but we have to filter the untrusted forks executions to avoid confusion.

Reply:
I will change the `on` trigger to `workflow_dispatch` and `schedule` only.
I think in that case we are good right?

Reply:
https://github.com/elastic/apm-pipeline-library/pull/1986/commits/b2b56bea4624c9dcf51a316c125ae58b7530f427